quickstart/001_client_server_introduction/README.md

```diff
@@ -0,0 +1,413 @@
+# Quickstart Tutorial - Introduction To VMware GemFire
+
+VMware GemFire is an in-memory distributed Key-Value datastore.  As a datastore, GemFire provides a real-time, consistent and distributed service for modern applications with data-intensive needs and low latency response requirements. Because of GemFire's distributed peer-to-peer nature it can take advantage of multiples servers to pool memory, cpu and disk storage for improved performance, scalability and fault tolerance to build applications needing caching, management of in-flight data or the key-value database of record.
+
+## Goal
+
+The goal of this quickstart tutorial is to introduce GemFire basics, including starting a GemFire development environment, building and connecting a Java based GemFire client application to a cluster and performing basic CRUD operations.
+
+## Download Examples and Configure Environment
+
+Download and install VMware GemFire from [Tanzu Network](https://network.pivotal.io). Follow the installation instructions in the [GemFire documentation](https://docs.vmware.com/en/VMware-Tanzu-GemFire/index.html).
+
+Clone the GemFire examples repository from GitHub.
+
+```text
+$ git clone git@github.com:gemfire/gemfire-examples.git
+```
+
+Set Gemfire home environment variable to top of GemFire install directory. Note for this example Gemfire is installed in the home directory of the user - adjust as necessary for local environment and install directory location.
```

Comment:
"Set Gemfire home environment variable to top..." -> "Set the GEMFIRE_HOME environment variable to the top..."